proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -89,6 +90,27 @@ enum PythonPackageType {
   PYTHON_PACKAGE_TYPE_STUB_ONLY = 2;
 }
 
+// DotnetTargetFramework specifies the target framework for dotnet plugins.
+// This is not meant to be an exhaustive list - most plugins should target a
+// netstandard version.
+// Reference: https://learn.microsoft.com/en-us/dotnet/standard/frameworks
+enum DotnetTargetFramework {
```

Comment:
How often is this going to change? This might be better as a dynamic string field if it will change often

Reply:
I don't think we'll need to change it very often at all - most plugins will require a pretty old version of this for compatibility with the broadest number of .NET runtimes.

Going forward we'd only add new values if they're required by a plugin.